workflows/Find_InsightConnect_Resources_from_Slack/workflow.spec.yaml

```
@@ -0,0 +1,27 @@
+extension: workflow
+products: ["insightconnect"]
+name: Find_InsightConnect_Resources_from_Slack
+title: "Find InsightConnect Resources from Slack"
+description: Find InsightConnect resources to support your initial implementation, without ever leaving Slack! Start using this workflow in minutes, without the need for an orchestrator.
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+hub_tags:
+  use_cases: [Chat, Slack]
```

```suggestion
  use_cases: []
```
We typically only use a set of use cases found [here](https://docs.google.com/document/d/1Z2wV5Z96jtxdz-LVsLzdl3S1aFo9RhVPODmylyJmay0) under the Workflow Tags --> Use Cases section, but I would say this workflow doesn't fit into any of those buckets